src/components/my-history/custom-my-history.vue

```diff
@@ -22,6 +22,16 @@
 
             <div class="x-flex x-flex-col x-gap-2">
               <p>{{ suggestion.query }}</p>
+              <ul v-if="suggestion.selectedFilters" class="x-flex x-gap-8">
```

Comment:
Doesn't look good when there are long/a lot of filters

![image](https://github.com/empathyco/x-archetype/assets/7124620/afc8439e-e9d3-40e8-8394-039c0907de2b)